src/explore-education-statistics-admin/src/pages/admin-dashboard/components/NonScheduledReleaseSummary.tsx

```diff
@@ -0,0 +1,104 @@
+import ButtonLink from '@admin/components/ButtonLink';
+import ReleaseSummary from '@admin/pages/admin-dashboard/components/ReleaseSummary';
+import { getReleaseSummaryLabel } from '@admin/pages/release/util/releaseSummaryUtil';
+import { summaryRoute } from '@admin/routes/edit-release/routes';
+import { AdminDashboardRelease } from '@admin/services/dashboard/types';
+import Button from '@common/components/Button';
+import { formatTestId } from '@common/utils/test-utils';
+import React from 'react';
+
+interface Props {
+  release: AdminDashboardRelease;
+  onClickAmendRelease?: (releaseId: string) => void;
+  onClickCancelAmendment: (releaseId: string) => void;
+}
+
+const NonScheduledReleaseSummary = ({
+  release,
+  onClickAmendRelease,
+  onClickCancelAmendment,
+}: Props) => {
+  return (
+    <ReleaseSummary
+      release={release}
+      actions={
+        <>
+          {release.amendment ? (
+            <>
+              <ButtonLink
+                to={summaryRoute.generateLink({
+                  publicationId: release.publicationId,
+                  releaseId: release.id,
+                })}
+                testId={formatTestId(
```

Comment:
Why do we need `formatTestId`? Doesn't seem like we have any line breaks in the string. Same for other `testId`s in this file.

Reply:
Yeah I dunno.  Was carrying over existing code when breaking out this new component.  Removed.